### PR TITLE
Amend language-models-and-dataset.md

### PR DESCRIPTION
**Pull request by cuongvng**
*Description of changes:*

A minor correction.

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

File: chapter_recurrent-neural-networks/language-models-and-dataset.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ bigram_vocab = d2l.Vocab(bigram_tokens)
 print(bigram_vocab.token_freqs[:10])
 ```
 
-Two things are notable. Out of the 10 most frequent word pairs, 9 are composed of stop words and only one is relevant to the actual book---"the time". Furthermore, let us see whether the trigram frequency behaves in the same manner.
+There is one thing notable here. Out of the 10 most frequent word pairs, 9 are composed of stop words and only one is relevant to the actual book---"the time". Furthermore, let us see whether the trigram frequency behaves in the same manner.
```
Comment (astonzhang) [inline chapter_recurrent-neural-networks/language-models-and-dataset.md:127]: one notable thing?

Comment (cuongvng) [reply chapter_recurrent-neural-networks/language-models-and-dataset.md:127]: I think so. It is just mentioned in the subsequent sentence:
> Out of the 10 most frequent word pairs, 9 are composed of stop words and only one is relevant to the actual book---"the time".

Comment (astonzhang) [reply chapter_recurrent-neural-networks/language-models-and-dataset.md:127]: I mean, one thing notable -> one notable thing

Comment (astonzhang) [reply chapter_recurrent-neural-networks/language-models-and-dataset.md:127]: or something notable

**Pull request merged**